### ISSUE TITLE
tcmalloc: large alloc 140113840283648 

### ISSUE THREAD

**Issue by poojahoza** [googlecolab/colabtools#2774]
**Describe the current behavior**
The script gives the following error on execution. It was working fine before.

**Describe the expected behavior**
The script was running smoothly without any issues previously. Expected it to run without any issues this time as well.

**What web browser you are using**
Chrome

**Additional context**
tcmalloc: large alloc 140113840283648 bytes == (nil) @  0x7f6eccf3b887 0x7f6ecb831c29 0x7f6ecb832afb 0x7f6ecb83320c 0x7f6e4d21b028 0x7f6e4d21eee8 0x7f6dda48f9aa 0x7f6dda48b352 0x7f6ecd1868d3 0x7f6ecd18b39f 0x7f6ecbe6916f 0x7f6ecd18a96a 0x7f6eccae3f96 0x7f6ecbe6916f 0x7f6ecbe691ff 0x7f6eccae4745 0x7f6eccae4051 0x7f6ecabc2a36 0x560e466f40e7 0x560e466f3ef0 0x560e467679a3 0x560e46762cdd 0x560e466f5f21 0x560e46737239 0x560e46734184 0x560e466f4902 0x560e46768522 0x560e466f57aa 0x560e46767d30 0x560e46762a2e 0x560e46762723
terminate called after throwing an instance of 'std::bad_alloc'
  what():  std::bad_allo
![google-colab-issue](https://user-images.githubusercontent.com/31398685/165349222-2c082bc5-0146-4c3b-a716-d607e033249b.png)
c

**Comment by craigcitro** [comment]
There's no repro info here, so it's hard to say, but I'm guessing you're hitting https://github.com/pyg-team/pytorch_geometric/issues/4539.

**Comment by poojahoza** [comment]
Thanks for sharing the issue, I tried the fix of using pytorch 1.11 but I am still getting the same issue. The code that I am using is as follows:

!pip install -q torch-scatter -f https://data.pyg.org/whl/torch-1.11.0+cu113.html
!pip install -q torch-sparse -f https://data.pyg.org/whl/torch-1.11.0+cu113.html
!pip install -q git+https://github.com/pyg-team/pytorch_geometric.git

from torch_geometric.data import Data
from torch_geometric.datasets import Planetoid
import torch_geometric.transforms as T

name_data = 'Cora'
dataset = Planetoid(root= '/tmp/' + name_data, name = name_data)
dataset.transform = T.NormalizeFeatures()
![google-colab-issue-2](https://user-images.githubusercontent.com/31398685/165563772-56a011bb-6f3b-4b70-a6d5-be93f95808d8.png)

**Comment by craigcitro** [comment]
We don't know anything about pytorch-geometric; you're going to have the most success asking upstream on their repo.

**Comment by RaM2VaR** [comment]
I am also facing the issue. I did not face this issue earlier. I was able to run the code in google colab successfully. But from yesterday onwards facing this issue.

**Comment by Shivam-Miglani** [comment]
@hosungs popping the env variable worked for me. I am using Google Colab pro which has 32GB RAM, but I get the same error when I try to allocate ~8GB at once. Unsetting the env var, I was able to use even 20GB at once.
